### DISCUSSION TITLE
DynamicForm Released 1.0.2

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#2236] [Show and tell]
Available for download through Oqtane Repository or Nuget
New Control added to the Toolbox {Tags}
![TagControl](https://user-images.githubusercontent.com/9447612/172578640-8929e47e-31c8-42a5-ae07-bcd438caa42b.gif)

Form Designer
![FormDesigner](https://user-images.githubusercontent.com/9447612/172578473-68c76f38-57fb-4a14-8165-479d34d12bd3.gif)

Presenter 
![Presenter](https://user-images.githubusercontent.com/9447612/172578507-20317336-b12f-4232-a87d-687d3d4c3b60.gif)

**Comment by bearcat9** [comment]
I added via nuget and I can see where i can add the module to a page however I am getting lots of 404 on resources for that module.  Its looking in a Modules/StudioElf.DynamicForm/  location however that doesn't exist in the project anywhere.  Is this something I need to create or should the nuget installation created this for me?

**Reply by leigh-pointer** [reply]
Hi @bearcat9 
This is very strange.  Would you mind installing the module from the Module Management inside your Oqtane instance.  Click the Install module button
![image](https://user-images.githubusercontent.com/9447612/172987003-1b857916-7fd6-4300-b53b-bb110f003778.png)
And download from here
![image](https://user-images.githubusercontent.com/9447612/172987081-7f6f359b-9923-4706-afa8-6435eef849d4.png)


You will need to restart you Oqtane Instance.

When Oqtane is fired up you should see the DynamicForm module ready for use in the Control Panel
![image](https://user-images.githubusercontent.com/9447612/172987375-2c27bd1e-5b4d-4a67-8d8b-c6f566195152.png)


Please, please let me know how you get on.

**Comment by bearcat9** [comment]
Once installed from the Oqtane repo it worked as expected kind of weird
that the night package didn’t work but oh well, not a big deal for me at
all.
